File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/response/uploadImage/UploadImageResultResponse.java
package com.lattis.ellipse.data.network.model.response.uploadImage;

import com.google.gson.annotations.SerializedName;

/**
 * Created by Velo Labs Android on 06-04-2017.
 */

public class UploadImageResultResponse {
    @SerializedName("uploaded_url")
    String uploaded_url;

    public String getUploaded_url() {
        return uploaded_url;
    }



    @Override
    public String toString() {
        return "UploadImageResultResponse{" +
                "uploaded_url=" + uploaded_url +
                '}';
    }
}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/response/uploadimage/UploadImageResultResponse.kt
package com.lattis.data.entity.response.uploadimage

import com.google.gson.annotations.SerializedName

class UploadImageResultResponse {

    @SerializedName("uploaded_url")
    var uploaded_url: String? = null

    override fun toString(): String {
        return "UploadImageResultResponse{" +
                "uploaded_url=" + uploaded_url +
                '}'
    }
}